test/functional/tests/conftest.py

```diff
@@ -164,19 +167,30 @@ def base_prepare(item):
                 casadm.stop_all_caches()
                 casadm.remove_all_detached_cores()
             except Exception:
-                pass  # TODO: Reboot DUT if test is executed remotely
+                TestRun.executor.reboot()
+            if without_installation:
+                installer.uninstall_opencas()
 
         for disk in TestRun.dut.disks:
             disk.umount_all_partitions()
             if not create_partition_table(disk, PartitionTable.gpt):
                 raise Exception(f"Failed to remove partitions from {disk}")
 
-        if get_force_param(item) and not TestRun.usr.already_updated:
+        if get_force_param(item) and not TestRun.usr.already_updated and not without_installation:
             installer.reinstall_opencas()
-        elif not installer.check_if_installed():
+
+        elif not installer.check_if_installed() and not without_installation:
             installer.install_opencas()
+
         TestRun.usr.already_updated = True
         TestRun.LOGGER.add_build_info(f'Commit hash:')
         TestRun.LOGGER.add_build_info(f"{git.get_current_commit_hash()}")
         TestRun.LOGGER.add_build_info(f'Commit message:')
         TestRun.LOGGER.add_build_info(f'{git.get_current_commit_message()}')
+
+
+def add_marks(config):
+    config.addinivalue_line(
+        "markers",
+        "without_installation: don't install tested software and if already installed, uninstall it"
```

Comment:
How about `uninstall_cas`? `without_installation` sounds like installation is skipped, but